packages/lib/src/components/internal/Countdown/index.tsx

```diff
@@ -8,75 +12,55 @@ interface CountdownProps {
 }
 
 interface CountdownState {
-    startTime: Date;
-    endTime: Date;
     minutes: string;
     seconds: string;
 }
 
-class Countdown extends Component<CountdownProps, CountdownState> {
-    constructor(props) {
-        super(props);
-        const secondsFromNow = this.props.minutesFromNow * 60000;
-        const nowTime = new Date().getTime();
-        this.state = {
-            startTime: new Date(nowTime),
-            endTime: new Date(nowTime + secondsFromNow),
-            minutes: '-',
-            seconds: '-'
-        };
-    }
-
-    public static defaultProps = {
-        onTick: () => {},
-        onCompleted: () => {}
-    };
-
-    protected interval;
+function Countdown({ minutesFromNow, onTick = () => {}, onCompleted = () => {} }: CountdownProps) {
+    const secondsFromNow = minutesFromNow * 60000;
+    const nowTime = new Date().getTime();
+    const startTime = new Date(nowTime);
+    const endTime = new Date(nowTime + secondsFromNow);
 
-    tick() {
-        const newTime = getTimeDifference(this.state.startTime, this.state.endTime);
+    const { i18n } = useCoreContext();
+    const { srPanel } = useSRPanelContext();
 
-        if (newTime.completed) {
-            this.props.onCompleted();
-            return this.clearInterval();
-        }
+    const a11yManager = useRef(null);
+    const [time, setTime] = useState<CountdownState>({
+        minutes: '-',
+        seconds: '-'
+    });
 
-        const time = {
-            minutes: newTime.minutes,
-            seconds: newTime.seconds,
-            percentage: newTime.percentage
+    useEffect(() => {
+        const tick = () => {
+            const { minutes, seconds, percentage, completed } = getTimeDifference(startTime, endTime);
+            if (completed) {
+                onCompleted();
+            }
+            const timeLeft = { minutes, seconds, percentage };
+            setTime(timeLeft);
+            onTick(timeLeft);
         };
+        a11yManager.current = new A11yManager({ srPanel, i18n });
```

Comment:
If I am not mistaken, you can do this at the beginning of this file. Better for readability as this needs to be created only once
```js
const a11yManager = useRef(new A11yManager({ srPanel, i18n }));
```

Reply:
See [my comment](https://github.com/Adyen/adyen-web/pull/2182#discussion_r1211793950)